index.html

```diff
@@ -5,7 +5,7 @@
     <meta charset="utf-8" />
     <meta name="viewport"
         content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui" />
-    <title>🎵 Daily Song</title>
+    <title>🎵 <%= title %></title>
```

Comment:
The emoji should be considered part of the configurable title, not every game will want this symbol

Reply:
We're still not focusing on game style customization, we can revisit this when we get back to this topic (or I can remove the emoji)